### ISSUE TITLE
post-session view update

### ISSUE THREAD

**Issue by JeffAbrahamson** [transport-nantes/tn_web#993]
Rather than going to a thank you page, we should send the person to the session page (which might still have a toast saying "awesome, well done".  But that page is then directly shareable.

The session page, for everyone, should have a button for making new sessions.  We want this to be viral.

**Comment by JeffAbrahamson** [comment]
Remaining:
* Remove "thanks" page if no longer needed.
* Modifications to session page to encourage sharing.
* A thank-you on the session page (e.g., post with an extra "thank-you" argument so that the argument isn't present when shared).

**Comment by Shriukan33** [comment]
How do you want to encourage sharing ? I think we usually avoid twitter/FB/Google links on our website. Although the integration here would make sense, but I think this would force us to have a cookie banner ?

**Comment by JeffAbrahamson** [comment]
> How do you want to encourage sharing ? I think we usually avoid twitter/FB/Google links on our website. Although the integration here would make sense, but I think this would force us to have a cookie banner ?

Twitter is easy enough, it can be done with a GET.
Facebook is harder, but a bit of searching proposes solutions.

This jQuery plugin looks promising:
https://panzi.github.io/SocialSharePrivacy/

[This article](https://www.business2community.com/social-media-articles/why-most-share-buttons-go-against-your-privacy-policy-02199051) suggests that (in some contexts, at a certain point in time) 2/3 of users shared via copy/paste rather than share buttons.  But share buttons remain important.

**Comment by JeffAbrahamson** [comment]
> > A thank-you on the session page (e.g., post with an extra "thank-you" argument so that the argument isn't present when shared).
> 
> Like a django message, it would be displayed once and never again ?

Yup.  Post to the URL rather than GET it, just so the arg isn't in the URL.  The arg has the thank you.  You don't even need a different handler for GET vs POST, you just check for the arg.  You don't care how it arrives.

**Comment by Shriukan33** [comment]
I added the plugin to see how it looks, and damn it shows it's not been updated for 10 years already : 
![image](https://user-images.githubusercontent.com/70256364/204299365-8654c293-5c16-48af-885c-a0dded3ea4ce.png)

However it does the job, no resources fetch to FB, Twitter & co. Clicking on it enables the button and we can share on the different platforms.

I'm going to see if there is a way to have more modern images.

**Comment by Shriukan33** [comment]
The inline version show better, I removed most of icons to keep it light : 
![image](https://user-images.githubusercontent.com/70256364/204305413-7faa77e0-fd9f-46fd-94b5-a013b7bf03c6.png)

**Comment by JeffAbrahamson** [comment]
Cool, I was about to suggest that we fork and adapt. ;-)

**Comment by JeffAbrahamson** [comment]
We can update the icons, too.  Maybe use fa?

**Comment by Shriukan33** [comment]
> We can update the icons, too. Maybe use fa?

Ha, didn't catch this comment before opening the PR.
I've seen dummy_img setting on the readme, will see to make it use the FA icons

**Comment by Shriukan33** [comment]
I can change the img's tag src, but that's about it. 
I couldn't get something ok to look at (or better than what is default.)

**Comment by JeffAbrahamson** [comment]
How do other sites we respect do this?

For example, [amnesty](https://www.amnesty.fr/refugies-et-migrants/actualites/naufrage-meurtrier-dans-la-manche-un-an-apres) just has this bit of html (on the page I linked):

`https://twitter.com/intent/tweet?text=Naufrage%20meurtrier%20dans%20la%20Manche%C2%A0%3A%20un%20an%20apr%C3%A8s%2C%20o%C3%B9%20en%20est-on%C2%A0%3F&url=https://www.amnesty.fr/refugies-et-migrants/actualites/naufrage-meurtrier-dans-la-manche-un-an-apres`

If a privacy blocker blocks something we do, it's a sign and risks reputational damage.  We simply shouldn't do it.

**Comment by Shriukan33** [comment]
Ok, will see to create our own buttons instead of this lib, which is obviously flawed in some ways.
